base/experiment.go

```diff
@@ -823,12 +833,13 @@ func BuildExperiment(withResult bool, driver Driver) (*Experiment, error) {
 			return nil, err
 		}
 	}
+	e.numLoops = 1
 	return &e, nil
 }
 
 // RunExperiment runs an experiment
-func RunExperiment(driver Driver) error {
-	if exp, err := BuildExperiment(false, driver); err != nil {
+func RunExperiment(driver Driver, reuseResult bool) error {
```

Comment:
Should I reverse the order of these arguments to match `BuildExperiment()`? Also what about the name of the argument? `reuseResult` vs `withResult` (like in in `BuildExperiment()`)? I feel like the former makes more sense in this context.